shared/src/commonMain/kotlin/org/hyperskill/app/network/NetworkBuilder.kt

```diff
@@ -1,4 +1,4 @@
-package org.hyperskill.app.network.injection
+package org.hyperskill.app.network
```

Comment:
Why moved from the `injection` package?

Reply:
What about this? Otherwise `expect / actual fun PreconfiguredPlatformHttpClient` not working?

Reply:
The same as with HttpBuilder.
Technically there is no problem to move it to `injection` package.
I just expect to find only `SomeModuleComponent` and it's implementation inside `injection` package.